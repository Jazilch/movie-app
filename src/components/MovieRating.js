/* eslint-disable camelcase */

'use es6';

import React from 'react';
import { CircleProgress } from 'react-gradient-progress';

const MovieRating = ({ vote_average }) => {
  const ratingPercentage = vote_average * 10;
  return (
    <CircleProgress percentage={ratingPercentage} width={70} fontSize="14" />
  );
};

export default MovieRating;
