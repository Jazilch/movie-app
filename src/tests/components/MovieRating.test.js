/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MovieRating from '../../components/MovieRating';

describe('MovieRating component', () => {
  it('renders the MovieRating correctly', () => {
    const movierating = shallow(<MovieRating />);
    expect(toJson(movierating)).toMatchSnapshot();
  });
});
