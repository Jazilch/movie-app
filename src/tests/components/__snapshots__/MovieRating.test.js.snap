// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieRating component renders the MovieRating correctly 1`] = `
<CircleProgress
  fill="transparent"
  fontColor="inherit"
  fontFamily="inherit"
  fontSize="14"
  percentage={NaN}
  primaryColor={
    Array [
      "#00BBFF",
      "#92d7f1",
    ]
  }
  secondaryColor="transparent"
  strokeWidth={5}
  width={70}
/>
`;
